conanfile.py

```diff
@@ -18,18 +20,19 @@ def configure(self):
 
     def build(self):
         args = []
-        if self.scope.build_test or self.scope.build_all:
+        if 'build_test' in os.environ or 'build_all' in os.environ:
             args += ["-DBUILD_TEST=1"]
-        if self.scope.build_server or self.scope.build_all:
+        if 'build_server' in os.environ or 'build_all' in os.environ:
             args += ["-DBUILD_SERVER=1"]
-        if self.scope.build_master or self.scope.build_all:
+        if 'build_master' in os.environ or 'build_all' in os.environ:
```

Comment:
These two lines can be omitted since there is no BUILD_MASTER anymore.

Reply:
Will do in further PRs.